tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -54,6 +54,19 @@ sub run {
             '/tmp/results/',
             '-type', 'f',
             '-iname', "*.xml");
+        my $find_ansible_know_issue = qr/\[OSADO\]\[softfail\] bsc#(\d+) (.*)/;
+        my $cat_command = "cat '$ret[1]'";
+        my $ansibleOutput = script_output($cat_command);
+        my ($bug_id, $description) = ("", "");
```

Comment:
Since `$bug_id` and `$description` are only used once, you can declare them in L65 & L66 respectively.

Reply:
refactored